sagan/train.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import tensorflow as tf
+from tensorflow import keras
+
+from models.generator import build_generator
+from models.discriminator import build_discriminator
+
+
+## Define the hyperparameters
+BATCH_SIZE = 64
+LATENT_DIM = 128
+EPOCHS = 100
+generator_save_path = "./saved_models/generator.h5"
+discriminator_save_path = "./saved_models/discriminator.h5"
+
+## Save the images
```

Comment:
utilities should ideally be in a separate file such as `utils.py`. Refer to https://github.com/soumik12345/point-cloud-segmentation/blob/main/point_seg/utils.py